diff --git a/node_modules/react-native-universal-pedometer/android/build.gradle b/node_modules/react-native-universal-pedometer/android/build.gradle
index 37145db..21692b4 100644
--- a/node_modules/react-native-universal-pedometer/android/build.gradle
+++ b/node_modules/react-native-universal-pedometer/android/build.gradle
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 22
+        targetSdkVersion 30
         versionCode 1
         versionName "1.0"
         ndk {
@@ -18,6 +18,11 @@ android {
     }
 }
 
+repositories {
+    mavenCentral()
+    google()
+}
+
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
