diff --git a/node_modules/react-native-accurate-step-counter/android/build.gradle b/node_modules/react-native-accurate-step-counter/android/build.gradle
index 22fc78f..51c635b 100644
--- a/node_modules/react-native-accurate-step-counter/android/build.gradle
+++ b/node_modules/react-native-accurate-step-counter/android/build.gradle
@@ -1,4 +1,3 @@
-
 buildscript {
     repositories {
         jcenter()
@@ -13,12 +12,12 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 22
+        targetSdkVersion 30
         versionCode 1
         versionName "1.0"
     }
@@ -33,6 +32,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
-  
\ No newline at end of file
